### PR TITLE
Add description for espresso testOptions object

### PR DESCRIPTION
**Pull request by valentyn88**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: docs/testrunner-toolkit/configuration/espresso.md [hunk 1]
```diff
@@ -103,3 +103,81 @@ __Example__:
...
+__Type__: *string*
+
+__Example__:
+```yaml
+  package: com.example.android.testing.androidjunitrunnersample
+```
+
+### `annotation`
+
+__Description__: Field for defining filter by selected annotation.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/espresso.md:176]:
```suggestion
__Description__: Instructs `saucectl` to run only tests that match a custom annotation that you have set. 
```

**Review by sweeneyskirt-sl — CHANGES_REQUESTED** [review]
I have made a few suggestions for clarity. @vrunoa -- Valentyn says you are the Android expert. Would you mind taking a look to make sure the descriptions are clear and "androidy"? ;)

File: docs/testrunner-toolkit/configuration/espresso.md [hunk 2]
```diff
@@ -103,3 +103,81 @@ __Example__:
     - "11.0"
     - "10.0"
 ```
+
+## `testOptions`
+
+__Description__: Field for defining filters for tests such as the class, size, package, notClass, and 
+annotation.
```
[the remark this answers is not in the document]
Comment (valentyn88) [reply docs/testrunner-toolkit/configuration/espresso.md]: done

File: docs/testrunner-toolkit/configuration/espresso.md [hunk 3]
```diff
@@ -103,3 +103,81 @@ __Example__:
...
+
+__Example__:
+```yaml
+  notClass:
+    - com.example.android.testing.androidjunitrunnersample.CalculatorInstrumentationTest
+```
+
+### `size`
+
+__Description__: Field for defining filter by specific test size.
```
[the remark this answers is not in the document]
Comment (valentyn88) [reply docs/testrunner-toolkit/configuration/espresso.md]: done

File: docs/testrunner-toolkit/configuration/espresso.md [hunk 4]
```diff
@@ -103,3 +103,81 @@ __Example__:
...
+  notClass:
+    - com.example.android.testing.androidjunitrunnersample.CalculatorInstrumentationTest  
+  size: small
+  package: com.example.android.testing.androidjunitrunnersample
+  annotation: com.android.buzz.MyAnnotation
+```
+
+### `class`
+
+__Description__: Field for defining filter by selected classes.
```
[the remark this answers is not in the document]
Comment (valentyn88) [reply docs/testrunner-toolkit/configuration/espresso.md]: done

File: docs/testrunner-toolkit/configuration/espresso.md [hunk 5]
```diff
@@ -103,3 +103,81 @@ __Example__:
...
+__Values__: *small*, *medium*, *large*
+
+__Example__:
+```yaml
+  size: small
+```
+
+### `package`
+
+__Description__: Field for defining filter by selected package.
```
[the remark this answers is not in the document]
Comment (valentyn88) [reply docs/testrunner-toolkit/configuration/espresso.md]: done

File: docs/testrunner-toolkit/configuration/espresso.md (hunk 1 above)
Comment (valentyn88) [reply docs/testrunner-toolkit/configuration/espresso.md]: done

**Review by sweeneyskirt-sl — APPROVED** [review]

**Pull request merged**